hollow/src/main/java/com/netflix/hollow/api/client/HollowClientUpdater.java

```diff
@@ -151,12 +151,19 @@ public synchronized boolean updateTo(long requestedVersion) throws Throwable {
             if (updatePlan.isSnapshotPlan()) {
                 HollowDataHolder oldDh = hollowDataHolderVolatile;
                 if (oldDh == null || doubleSnapshotConfig.allowDoubleSnapshot()) {
-                    // The volatile field needs to be assigned here because it may be accessed when
-                    // executing the update plan, for example a refresh listener may access the consumer
-                    // after the plan has been applied (such as a unique key indexer).
```

Comment:
What about this comment that's getting deleted -- it seems like a valid point?
Under the `HollowDataHolder.update()` method, calls to the refreshListeners are made, right?

Reply:
Yes, this is expected - the comment below tries to capture that it maintains the same invariant. The short version is that we want to be sure that the data holder has been swapped before we call any listeners. That is still the case in this PR.